### PR TITLE
Mention documentation improvements

### PR DESCRIPTION
**Pull request by JackStouffer**
Reopened on stable.

### CODE REVIEW

**Comment by JackStouffer** [comment]
ping @MartinNowak

File: changelog/2.069.0_pre.dd [hunk 1]
```diff
@@ -188,6 +189,14 @@ $(LI $(LNAME2 curl-dynamic-loading, libcurl is now loaded dynamically)
     )
 )
 
+$(LI $(RELATIVE_LINK2 documentation-improvements, The documentation for Phobos has been improved)
+
+    $(P A large portion of the functions in Phobos have improved documentation, including but
+        not limited to spelling fixes, clarifications, better explainations of function and template
```
Comment (CyberShadow) [inline changelog/2.069.0_pre.dd]: s/explainations/explanations/g

**Comment by CyberShadow** [comment]
What's the corresponding bugzilla issue / forum discussion?

**Comment by JackStouffer** [comment]
@CyberShadow Huh?

**Comment by CyberShadow** [comment]
Well, like, is there any discussion or specifics of what this changelog entry is documenting? I seem to have missed the great documentation improvement effort described here.

**Comment by JackStouffer** [comment]
Well, quickfur, burner, and I have a combined 40+ PRs just on Phobos docs since the last release.

**Comment by CyberShadow** [comment]
OK.

**Comment by CyberShadow** [comment]
Ping?

**Comment by JackStouffer** [comment]
@CyberShadow my git skills aren't good enough to fix this. I'll most likely end up opening a new PR based off of this later today.

**Pull request closed without merging**